dist/opt/flight/libexec/commands/set

```diff
@@ -133,9 +133,25 @@ case $action in
   DESCRIPTION:
 
     Set the ${flight_STARTER_product} setting <key> to <value>. Valid
-    keys are 'hints', 'welcome', 'secondary' and 'always'. Valid
+    keys are 'hints', 'welcome', 'secondary', 'always', 'tips' and 'status'. Valid
     values are 'on' and 'off'.
 
+    The keys are as follows: 
+
+      hints         Toggle showing of hints to start/enable ${flight_STARTER_product}
+                    system (when 'always' is off) 
+
+      welcome       Toggle showing of welcome to cluster line (when 'always' is off)
+
+      secondary     Toggle running of ${flight_STARTER_product} on secondary shells
+
+      always        Set whether ${flight_STARTER_product} is activated on login 
+
+      tips          Set ${flight_STARTER_product} sub-command tips (when 'always' is on)
+
+      status        Set showing of ${flight_STARTER_product} cluster status
+                    (when 'always' is on)
```

Comment:
Why is this only active when `always` is on? Doesn't the status (and tips) appear when `flight start` is executed if `always` isn't on?

Reply:
Good point, I will adapt to mention for "active flight environment" or something to imply when `flight start` is run or `always on` is set